### PR TITLE
Node api fix dao

### PR DESCRIPTION
**Pull request by boqnd**

### CODE REVIEW

File: server/dao/games.js [hunk 1]
```diff
@@ -13,11 +15,25 @@
...
  * get game
- * @param {string} id - game id
+ * @param {string} gameId - game id
  * @returns {object} - object containing user ids and game state
  */
-function getGame(id) {
-  // TODO:
+function getGame(gameId, token) {
+  // TODO: remove auth for this
+  const url = `http://localhost:8080/state/${gameId}/currentState`;
```
Comment (BlagovestMihaylov) [inline server/dao/games.js:23]: this wont work in docker 
(localhost)

File: server/dao/games.js [hunk 2]
```diff
@@ -26,20 +42,61 @@ function getGame(id) {
...
-function updateGame(id, data) {
-  // TODO:
+function updateGame(gameId, data) {
+  // TODO: playerId in body not needed
+  const url = `http://localhost:8080/state/update`;
+
+  let config = {
+    headers: {
+      'Content-Type': 'application/json',
+      'Authorization' : data.token
```
Comment (BlagovestMihaylov) [inline server/dao/games.js:82]: i suggest keeping a map with token per username and if it responds with 403 to refresh the map

Comment (BlagovestMihaylov) [reply server/dao/games.js:82]: also we need the bearer

File: server/dao/games.js (hunk 1 above)
Comment (boqnd) [reply server/dao/games.js:23]: I will add those strings in env file, for now this is fine

File: server/dao/games.js (hunk 2 above)
Comment (boqnd) [reply server/dao/games.js:82]: Ako ima vreme sled nakraq 🙂

**Review by BlagovestMihaylov — APPROVED** [review]

**Pull request merged**